agents/che-core-api-agent/src/test/resources/run_launcher_bats_tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
it could be run as part of the profile integration tests and using docker containers 
https://github.com/eclipse/che/blob/185273f8b92fe5523e676255cd7be172e90ee418/plugins/plugin-java-debugger/che-plugin-java-debugger-server/pom.xml#L234-L235